eclass/kernel-install.eclass

```diff
@@ -442,6 +595,22 @@ kernel-install_pkg_preinst() {
 	fi
 }
 
+# @FUNCTION: kernel-install_extract_from_uki
+# @USAGE: <directory> <output-file>
```

Comment:
This makes very little sense. Directory is not used for anything except for constructing `uki.efi`, and at least right now it's fully predictable from `out`. If we want to make it support output into a different directory (which makes sense), then make the first arg `uki.efi` patch, to have `<input> <output>`. And make `extract_type` an explicit argument instead of requiring predictable filenames.